### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: assets/css/structure.css [hunk 1]
```diff
@@ -17,7 +17,3 @@
 .fourth-section {
   border: 2px solid red;
 }
```
Comment (blacksrc) [inline assets/css/structure.css]: Why did you remove `.site-footer` class? You still have it inside the html?

File: index.html [hunk 2]
```diff
@@ -48,11 +48,9 @@
       </section>
     </div>
     <footer class="site-footer">
-      <div class="site-wrapper">
-        <div class="footer-container">
-          <div class="">1</div>
-          <div class="">2</div>
-        </div>
+      <div class="footer-container">
```
Comment (blacksrc) [inline index.html:64]: Where can I find  the class definition for `footer-container` in css?

**Review by takrami — APPROVED** [review]
Nice!

File: assets/css/structure.css (hunk 1 above)
Comment (hedita) [reply assets/css/structure.css]: Yeah I have it in html,should I remove it?

File: index.html (hunk 2 above)
Comment (hedita) [reply index.html:64]: I removed it.

Comment (hedita) [reply index.html:64]: In structure-desktop.

File: assets/css/structure.css (hunk 1 above)
Comment (blacksrc) [reply assets/css/structure.css]: No if you have it then it's fine.

File: index.html (hunk 2 above)
Comment (blacksrc) [reply index.html:64]: ok

**Review by blacksrc — APPROVED** [review]

**Review by blacksrc — APPROVED** [review]
